manifests/init.pp

```diff
@@ -4,7 +4,7 @@
 #
 # === Parameters:
 #
-# $parent_fqdn::                        FQDN of the parent node.
+# $parent_fqdn::                        FQDN of the parent pulp node.
```

Comment:
I further looked into this: there is no pulp parent, just the Foreman URL and the qpid hub. We can take the former from the proxy config. Not sure what to name the latter.